src/integration/integration_api_scheduler_test.go

```diff
@@ -70,6 +70,34 @@ var _ = Describe("Integration_Api_Scheduler", func() {
 				Expect(resp.StatusCode).To(Equal(http.StatusNotFound))
 				resp.Body.Close()
 			})
+
+			Context("Scheduler is down", func() {
+				BeforeEach(func() {
+					stopScheduler(schedulerProcess)
+
+					policyStr = readPolicyFromFile("fakePolicyWithSchedule.json")
+					resp, err := attachPolicy(appId, policyStr)
+					Expect(err).NotTo(HaveOccurred())
+					Expect(resp.StatusCode).To(Equal(http.StatusInternalServerError))
+					resp.Body.Close()
+
+					schedulerProcess = startScheduler()
```

Comment:
I was not clear in my explanation before. The startScheduler is part of the test case to verify the state of the data. Maybe better to have it in the "It" section. But still I understand we should have another start scheduler in the afterEach in case something goes wrong in the middle.